test/runner/requirements/integration.cloud.aws.txt

```diff
@@ -1,2 +1,3 @@
 boto
 boto3
+botocore>=1.10.30
```

Comment:
Requirements such as `botocore` should be listed in this file, but version constraints such as `botocore >= 1.10.30` should be in `constraints.txt` instead. Be sure to include a comment explaining the reason for the constraint.